tardis/simulation/base.py

```diff
@@ -626,33 +626,15 @@ def log_plasma_state(
         plasma_state_log["next_w"] = next_dilution_factor
         plasma_state_log.columns.name = "Shell No."
 
-        if is_notebook():
-            logger.info("\n\tPlasma stratification:")
-
-            # Displaying the DataFrame only when the logging level is NOTSET, DEBUG or INFO
-            if logger.level <= logging.INFO:
-                if not logger.filters:
-                    display(
-                        plasma_state_log.iloc[::log_sampling].style.format(
-                            "{:.3g}"
-                        )
-                    )
-                elif logger.filters[0].log_level == 20:
-                    display(
-                        plasma_state_log.iloc[::log_sampling].style.format(
-                            "{:.3g}"
-                        )
-                    )
-        else:
-            output_df = ""
-            plasma_output = plasma_state_log.iloc[::log_sampling].to_string(
-                float_format=lambda x: f"{x:.3g}",
-                justify="center",
-            )
-            for value in plasma_output.split("\n"):
-                output_df = output_df + f"\t{value}\n"
-            logger.info("\n\tPlasma stratification:")
-            logger.info(f"\n{output_df}")
+        output_df = ""
+        plasma_output = plasma_state_log.iloc[::log_sampling].to_string(
+            float_format=lambda x: f"{x:.3g}",
+            justify="center",
+        )
+        for value in plasma_output.split("\n"):
+            output_df = output_df + f"\t{value}\n"
+        # logger.info("\n\tPlasma stratification:")
```

Comment:
Why is this commented out?

Reply:
I added it back, the dataframe was not being rendered in the widget, I fixed that now.